include/clock.h

```diff
@@ -5,5 +5,6 @@
 
 void InitTimeBasedEvents(void);
 void DoTimeBasedEvents(void);
+void FormChangeTimeUpdate();
```

Comment:
Is there a reason behind this? If this means it is currently broken on master, it needs to be extracted and PR'd separately.

Reply:
Seems like a remmnant of it trying to use unused IDs. I'll make a new one instead like how I did followers

Reply:
To account for fake RTC, I moved `FormChangeTimeUpdate` from `UpdatePerMinute` to `OverworldBasic` so that form changes happen based on the accelerated time and not real time.